### ISSUE TITLE
AdvantageScope doesn't start in fresh Ubuntu 24.04 installation

### ISSUE THREAD

**Issue by sciencewhiz** [wpilibsuite/WPILibInstaller-Avalonia#416]
Starting Advantage scope on a fresh Ubuntu 24.04 installation prints the following error message about missing libfuse2.

```
dlopen(): error loading libfuse.so.2

AppImages require FUSE to run. 
You might still be able to extract the contents of this AppImage 
if you run it with the --appimage-extract option. 
See https://github.com/AppImage/AppImageKit/wiki/FUSE 
for more information
```

In Ubuntu24.04, the correct version of libfuse can be installed with `sudo apt install libfuse2t64`

Running with the appimage-extract does extract the image, so that could be done by the installer, and the script could be updated to call the extracted location.

Once it is extracted, it fails to run for the chrome-sandbox (same as #411 and #412 and #414).

**Comment by spacey-sooty** [comment]
Another option would be installing it through the deb on debian based distros

**Comment by sciencewhiz** [comment]
Installing the .deb or other package would require root.

**Comment by sciencewhiz** [comment]
Fixed by #424
